### PR TITLE
some vague barrier stats

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,8 +6,7 @@ language: node_js
 node_js:
 - 0.12.*
 after_success:
-- test $TRAVIS_PULL_REQUEST == "false" && test $TRAVIS_BRANCH == "master" && make
-  deploy
```
Comment (aintgoin2goa) [inline .travis.yml]: thought it would be a good idea to disable travis deploys for the time being?

Comment (matthew-andrews) [inline .travis.yml]: is there a problem?

Comment (commuterjoy) [inline .travis.yml]: I've fixed that I think. merge away.

Comment (matthew-andrews) [inline .travis.yml]: wait a tic, don't merge yet.  do we mean to disable this?

Comment (aintgoin2goa) [inline .travis.yml]: I did at the time, cos Travis deploys were breaking the beacon dashboard, but I think it's fine now

**Pull request merged**